### DISCUSSION TITLE
CSB for "playground" crashes

### DISCUSSION THREAD

**Discussion by zehawki** [igordanchenko/yet-another-react-lightbox#103] [General]
https://codesandbox.io/s/yet-another-react-lightbox-examples-9qvmif?file=/src/examples/Playground.js&initialpath=/examples/playground

When changing the Slides slider, I get:

![image](https://user-images.githubusercontent.com/8095144/229285714-c203a429-3c83-430e-8cc2-455f73830ebe.png)

**Comment by igordanchenko** [comment]
Thank you for catching this. I've updated the playground.
